tick/survival/tests/model_coxreg_partial_lik_test.py

```diff
@@ -24,6 +24,22 @@ def test_ModelCoxRegPartialLik(self):
         model_spars.fit(csr_matrix(features), times, censoring)
         self.run_test_for_glm(model, model_spars)
 
+    def test_sccs_serialize_and_compare(self):
+        """Test longitudinal multinomial model Lipschitz constant."""
+        np.random.seed(123)
+        n_samples, n_features = 100, 5
+        w0 = np.random.randn(n_features)
+        features, times, censoring = SimuCoxReg(
+            w0, n_samples=n_samples, verbose=False, seed=1234).simulate()
+        model = ModelCoxRegPartialLik()
+        model.fit(features, times, censoring)
+        model_spars = ModelCoxRegPartialLik()
+        model_spars.fit(csr_matrix(features), times, censoring)
+
```

Comment:
Why are you creating a sparse version if you do not use it in the test assertions?

Reply:
copy/pasta again, I just took the top test and changed it (not enough!)